tests/specs/scss/helpers/_import-once.test.scss

```diff
@@ -0,0 +1,21 @@
+@include test-module("import-once") {
+  // Testing char-at functions
+  @include test("import-once should include all modules only once.") {
+    $expect: ("test" "test-1");
+
+    @if import-once("test") {
+      .test {}
+    }
+
+    @if import-once("test-1") {
+      .test-1 {}
```

Comment:
Empty rule<br>Selector `test-1` should match regex /^(?:u|is|has)\-[a-z][a-zA-Z0-9]*$|^(?!u|is|has)[a-zA-Z][a-zA-Z0-9]*(?:\-[a-z][a-zA-Z0-9]*)?(?:\-\-[a-z][a-zA-Z0-9]*)?$/